Update readme with data tag and step icon guide (#27)
experiandataquality/aperture-data-studio-sdk: README.md

    }
}
```
##### Setting Step Icon

> As demonstrated above, step icon can be set via `setStepDefinitionIcon` method. 

Reviewer: ```suggestion
As demonstrated above, a custom step icon can be set via the `setStepDefinitionIcon` method. 
```

    .withSelectAllOption(false);
```

##### Auto select column based on data tag

> Data Studio allows tagging columns with additional label. Refer to [Data Tagging](https://www.edq.com/documentation/aperture-data-studio/user-guide/#data-tagging) for more info. 

Reviewer: ```suggestion
Data Studio allows columns to be tagged columns with an additional label. Refer to [Data Tagging](https://www.edq.com/documentation/aperture-data-studio/user-guide/#data-tagging) for more info. 
```

> You can make the column chooser step property to automatically select a column tagged with specific label (e.g. 'Phone') once the custom step connected with an input.

Reviewer: ```suggestion
The column chooser step property can automatically select a column tagged with specific label (e.g. 'Phone') once the custom step connected with an input.
```

        setStepDefinitionName("Custom - Column Tag");
        setStepDefinitionIcon("INFO");

        final StepProperty spPhone = new StepProperty()
                .ofType(StepPropertyType.COLUMN_CHOOSER)
                .withStatusIndicator(sp -> () -> isPhoneArgOk(sp))
                .withIconTypeSupplier(sp -> () -> isPhoneArgOk(sp) ? "OK" : "ERROR")
                .withArgTextSupplier(sp -> () -> getPhoneArgText(sp))
                .havingInputNode(() -> "input0")
                .havingOutputNode(() -> "output0")
                .validateAndReturn();
        setStepProperties(Arrays.asList(spPhone));

        setStepOutput(new ColumnTagOutput());
    }

    private boolean isPhoneArgOk(final StepProperty sp) {
        autoSelectTaggedColumn(sp, TAG_PHONE);
        return sp.allowedValuesProvider != null && sp.getValue() != null;
    }

    private String getPhoneArgText(final StepProperty sp) {
        autoSelectTaggedColumn(sp, TAG_PHONE);
        if (sp.allowedValuesProvider == null) {
            return "<Connect an input>";
        } else {
            return sp.getValue() == null ? "<Select a phone column>" : sp.getValue().toString();
        }
    }
```

> Code above is very similar to normal step configuration, except it will call `autoSelectTaggedColumn` method every time it updates UI with `withStatusIndicator`, `withIconTypeSupplier` and `withArgTextSupplier` methods.

Reviewer: ```suggestion
The code above is very similar to normal step configuration, except it will call the `autoSelectTaggedColumn` method every time it updates the UI with `withStatusIndicator`, `withIconTypeSupplier` and `withArgTextSupplier` methods.
```

```java

/**

> * Auto set step property's value with column tagged with specific name.

Reviewer: ```suggestion
* Automatically set step property's value to a column tagged with specific label name.
```
